src/app/component/header/header.component.html

```diff
@@ -52,6 +52,7 @@
                 class="btn btn-primary"
                 data-bs-toggle="modal"
                 data-bs-target="#exampleModal1"
```

Comment:
what is use for  data-bs-toggle="modal"
                data-bs-target="#exampleModal1"

Reply:
i removed it as it is not affecting anything